scapy/sendrecv.py

```diff
@@ -93,6 +95,11 @@ def _get_pkt():
         def _get_pkt():
             if bpf_select([pks]):
                 return pks.recv()
+    elif PYTHON_CAN:
```

Comment:
wrong. If PYTHON_CAN but not the isinstance, we lost leave the if

Reply:
Right. Thanks. Fixed it